docker/ubuntu_20_04_nolofar

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:20.04
+
+RUN export DEBIAN_FRONTEND=noninteractive && apt-get update && \
+    apt-get install -y \
+    libboost-python-dev libcfitsio-dev python3-dev python3-numpy cmake \
+    build-essential libhdf5-serial-dev libarmadillo-dev \
+    libboost-filesystem-dev libboost-system-dev libboost-date-time-dev \
+    libboost-program-options-dev libboost-test-dev \
+    libxml2-dev libpng-dev pkg-config \
+    libgtkmm-3.0-dev git libfftw3-dev \
+    gfortran flex bison wcslib-dev \
+    libboost-numpy-dev liblua5.3-dev \
+    casacore-dev aoflagger-dev
+
+ADD . /src
+WORKDIR /src
+
+RUN mkdir /build && cd /build && cmake ../src
+RUN cd /build && make -j2
+RUN cd /build && make install -j2
+RUN DPPP
+RUN cd /build && CTEST_OUTPUT_ON_FAILURE=1 make test -j2
```

Comment:
It could be better to chain all these commands into a single line. This can reduce the size of your container because it limits the amount of layers that are built.